api/v1/routes/testimonial.py

```diff
@@ -1,18 +1,48 @@
-from fastapi import Depends, HTTPException, APIRouter, Request, Response, status
+#!/usr/bin/env python3
+"""
+Module contains CRUD routes for testimonial
+"""
+from api.db.database import get_db
 from sqlalchemy.orm import Session
+from api.utils.dependencies import get_super_admin
+from api.v1.models.user import User
+from api.v1.models.testimonial import Testimonial
+from uuid import UUID
+from fastapi import Depends, HTTPException, APIRouter, Request, Response, status
 from fastapi.responses import JSONResponse
-
 from api.utils.success_response import success_response
-from api.v1.models.testimonial import Testimonial
-from api.v1.models.user import User
-# from api.v1.schemas.user import DeactivateUserSchema, UserBase
-from api.db.database import get_db
 from api.v1.services.testimonial import testimonial_service
 from api.v1.services.user import user_service
 
-
 testimonial = APIRouter(prefix='/testimonials', tags=['Testimonial'])
 
+@testimonial.delete("/{testimonial_id}")
+def delete_testimonial(
+    testimonial_id: UUID,
+    current_user: User = Depends(get_super_admin),
+    db: Session = Depends(get_db)
+):
+    """
+    Function for deleting a testimonial based on testimonial id
+    """
+    testimonial = db.query(Testimonial).filter(Testimonial.id == testimonial_id).first()
+    if not testimonial:
+        raise HTTPException(
+            detail="Testimonial not found",
+            status_code=status.HTTP_404_NOT_FOUND
+        )
+    db.delete(testimonial)
+    db.commit()
```

Comment:
create a file in services folder for your database interactions

Reply:
done!